### PR TITLE
update AM with D00025 CE02 and finalized D00024

### PR DESCRIPTION
**Pull request by lstoltz**
I will send file checks in an email.

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -284,4 +284,5 @@ EK20220510,Elakha,2022-05-10T16:30:00,2022-05-10T22:04:00,Endurance glider and C
 SKQ202208S,Sikuliaq,2022-05-11T00:00:00,2022-05-30T23:59:00,Papa 9
 TS20220518,Tenacious II,2022-05-18T11:00:00,2022-05-19T03:00:00,Pioneer 18a glider deployments
 TS20220526,Tenacious II,2022-05-26T01:00:00,2022-05-26T23:00:00,Pioneer 18b test glider deployments and cp_379 recovery
-AR69-01,Neil Armstrong,2022-06-20T00:00:00,2022-07-17T23:59:00,Irminger 9
\ No newline at end of file
+EK20220526,Elakha,2022-05-26T16:02:00,2022-05-26T19:04:00,Endurance CE02SHSP turn
```
Comment (cwingard) [inline cruise/CruiseInformation.csv:287]: You have the launch time of the CSPP as the end time of the cruise. Time should be 22:22 from the config sheet.

**Review by cwingard — CHANGES_REQUESTED** [review]
Need to update the cruise end time. Also, need to send me the nutnr cal file.

**Review by cwingard — APPROVED** [review]

**Comment by desiderr** [comment]
Verified serial numbers from pre-deployment test files, except for the ctd and velpt. Everything else appears to be in order except for a discrepancy in deployment latitude. The cruise report reports 44 degrees, 37.981 minutes, which comes out to 44.633017 degrees, instead of 44.633117 as in the pull request.

**Comment by lstoltz** [comment]
> Verified serial numbers from pre-deployment test files, except for the ctd and velpt. Everything else appears to be in order except for a discrepancy in deployment latitude. The cruise report reports 44 degrees, 37.981 minutes, which comes out to 44.633017 degrees, instead of 44.633117 as in the pull request.

I have the deployment latitude as 44 degrees  37.987 minutes which comes out to 44.633117. I checked the Al Fresco submission and saw the same results. Did I send you an incorrect form by mistake? If so please delete and refer to the correct version documented on Al Fresco. Thanks.

**Pull request merged**